empress/recon_vis/utils.py

```diff
@@ -62,6 +120,7 @@ def _dict_to_tree_helper(tree_dict, root_edge):
         new_node.left_node = new_left_node
         new_left_node.parent_node = new_node
         new_node.right_node = new_right_node
+        new_left_node.parent_node = new_node
```

Comment:
I think this is a duplicate code. Please remove this line.

Reply:
fixed